### PR TITLE
Release v3.2.0

### PR DESCRIPTION
**Pull request by mgrachev**

### CODE REVIEW

**Review by mstruebing — APPROVED** [review]
🙌

File: docs/whats_new/v320.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+### 2. Check for updates 🆕
+
+Now `dotenv-Linter` will check the availability of a new version once a day.<br/>
+If the new version is available, then `dotenv-Linter` will display this message:
+
+<img width="714" alt="colored_output" src="_assets/docs/whats_new/v320/check_for_updates.png">
+
+You can disable checking for updates using the `--not-check-updates` flag.
+
+PR: [#454](https://github.com/dotenv-linter/action-dotenv-linter/pull/454) ([@mgrachev](https://github.com/mgrachev))
```
Comment (DDtKey) [inline docs/whats_new/v320.md:34]: Links here are broken. It should be `https://github.com/dotenv-linter/dotenv-linter/....`

Comment (mgrachev) [reply docs/whats_new/v320.md:34]: Good catch 👀

Comment (mgrachev) [reply docs/whats_new/v320.md:34]: Fixed 🙏

**Pull request merged**